src/app/components/login/login.rs

```diff
@@ -32,6 +59,16 @@ impl Login {
                 model.login(username, password);
```

Comment:
The click handler could use the same handler and logic as below, if we rename it to something like `submit_login_form` or something, idk!

Reply:
@xou816 is 06df1a38df18457ab917f0c0182b150db72837f4 what you meant?